Makefile

```diff
@@ -1,7 +1,10 @@
 QDL := qdl
 RAMDUMP := qdl-ramdump
 
-CFLAGS += -O2 -Wall -g `pkg-config --cflags libxml-2.0 libusb-1.0`
+BUILD_TIME := $(shell date +"%c")
+VERSION := "1.0.0"
```

Comment:
Probably good to add git-version info (git describe) as well or instead.

Reply:
For the version information to be useful (e.g. for bug reports etc), we need to bump this version string on every commit. So, I think git describe provides much more value here.